api/src/main/java/com/codeforcommunity/dto/NewsReturn.java

```diff
@@ -8,16 +8,18 @@
   private final int id;
   private final String title;
   private final String description;
+  private final String imageUrl;
   private final String author;
-  private final LocalDateTime date;
+  private final String date;
   private final String content;
 
-  public NewsReturn(int id, String title, String description, String author, LocalDateTime date, String content) {
+  public NewsReturn(int id, String title, String description, String author, LocalDateTime date, String content, String imageUrl) {
     this.id = id;
     this.title = title;
     this.description = description;
+    this.imageUrl = imageUrl;
     this.author = author;
-    this.date = date;
+    this.date = date.toString();
```

Comment:
Will this give you the date in the format you specified in the API doc? It would be worth adding a comment (maybe as javadoc on the date field) of the format of string you're expecting.

Reply:
Currently our dates are a bit strange. Ideally we would take in (in Javascript) a new Date().toJson() which is a string formatted "yyyy-MM-ddTHH:mm:ss.SSSZ", and be able to parse that on the backend into a LocalDateTime, and then back into that format.